app/controllers/cart/cart_items_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Cart::CartItemsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    current_cart.add_cart_item(cart_item_params[:product_id], cart_item_params[:quantity])
```

Comment:
朝話した通り ↓ の方がよいかも
add_cart_item の中で find して 404 になるのは少し驚きがあるので

```suggestion
    product = Product.purchasable.find(cart_item_params[:product_id])
    current_cart.add_cart_item(product, cart_item_params[:quantity])
```

Reply:
[e72197e](https://github.com/kk0000-kk/sakura-market/pull/25/commits/e72197e3a200889a967d074953afef01d621a71c)